### PR TITLE
fludashboard added.

### PR DESCRIPTION
**Pull request by xmnlab**

### CODE REVIEW

**Comment by xmnlab** [comment]
@marfcg 

I added you as maintainer too :+1:

File: recipes/fludashboard/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win or py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
```
Comment (isuruf) [inline recipes/fludashboard/meta.yaml]: Do you need `pip` here?

**Comment by isuruf** [comment]
Looks good to me. If @marfcg confirms that they want to be a maintainer, I'll merge.

File: recipes/fludashboard/meta.yaml (hunk 1 above)
Comment (xmnlab) [reply recipes/fludashboard/meta.yaml]: I use pip in setup.py ..  When it tried to build for py2k raised error .. Now we just build for linux and osx and for py3k ..  If yoy want we can try again with no pip in requirements.

**Comment by xmnlab** [comment]
@isuruf 

@marfcg confirmed with a "like" emoticon in the comment: https://github.com/conda-forge/staged-recipes/pull/4258#issuecomment-339501150 ... (Just available on computer view mode).

**Comment by jakirkham** [comment]
No need to be shy @marfcg. 😉

**Comment by jakirkham** [comment]
Thanks @xmnlab and @isuruf.

**Comment by marfcg** [comment]
@jakirkham I've decided it's about time I should come out: I'm openly declaring that I'm IN :smile: 
Sorry for the delay and thanks for accepting our contribution.

**Pull request merged**